benchmark.py

```diff
@@ -30,7 +30,7 @@ def Record(scenario_name, driver_script, output_dir, browser=None, extra_args=[]
   if browser is not None:
     browser_executable = utils.browsers_definition[browser]['executable']
     if browser in ["Chrome", "Canary", "Edge"]:
-      subprocess.call(["open", "-a", browser_executable, "--args"] + extra_args)
+      subprocess.call(["open", "-a", browser_executable, "--args"] + ["--force-fieldtrials=BackgroundTracing/default83/", "--disable-stack-profiler"] + extra_args)
```

Comment:
Unfortunately the group for disabling changes often (as you can see it's at version 83) is there anything we can do to make sure it gets updated?

Reply:
It would be a **total** hack but we could at least force a dozen extra future revisions to make sure we don't hit this during this project.

Reply:
Added enable-benchmarking instead :)